swig/python/codegen/codegen.py

```diff
@@ -0,0 +1,1172 @@
+from __future__ import print_function, unicode_literals
+import six
+import sys
+import ast
+import os
+import tokenize
+import warnings
+from six import StringIO
+
+
+def interleave(inter, f, seq):
+    """Call f on each item in seq, calling inter() in between.
+    """
+    seq = iter(seq)
+    try:
+        f(next(seq))
+    except StopIteration:
+        pass
+    else:
+        for x in seq:
+            inter()
+            f(x)
+
+class CodeGenException(Exception):
+    """ Generic exception for errors raised in code generation """
+    pass
```

Comment:
This `pass` statement is not actually necessary, as docstrings count as an expression (they are not a comment, as they are become a member of the class). 

No harm in keeping it, just something odd that I found out recently.

Reply:
I think ast parsing dumps them though